urdf_model/include/urdf_model/utils.h

```diff
@@ -0,0 +1,67 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Willow Garage doesn't exist either. I don't know what to do about licensing for new files, so I have just copied the old license.

I'll cc @tfoote and @gerkey for some other opinions.